src/Lykke.Bil2.Contract/TransactionsExecutor/Requests/TransactionCoinsValidator.cs

```diff
@@ -38,7 +38,7 @@ public static void Validate(IReadOnlyCollection<CoinToSpend> coinsToSpend, IRead
                 .Select(g => new
                 {
                     AssetId = g.Key,
-                    Sum = g.Sum(x => x.Value)
+                    Sum = g.Select(x => x.Value).Aggregate((x, y) => x + y)
```

Comment:
Would be nice to have overloads of Sum() (and another Linq methods) which can work with Money/UMoney directly.